src/cascadia/TerminalApp/Profile.idl

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace TerminalApp
+{
+    enum CloseOnExitMode
+    {
+        Never = 0,
+        Graceful,
+        Always
+    };
+
+    [default_interface] runtimeclass Profile {
+        Profile();
+        Profile(Windows.Foundation.IReference<Guid> guid);

```

Comment:
just to be clear, then?

there's 3 ways to construct profile

1. no arg
2. nullptr
3. guid


the IReference doesn't need to be on the ctor. you _already have that_ by way of having two ctors.